components/utilities/rebind.js

```diff
@@ -0,0 +1,18 @@
+(function(d3, fc) {
+    'use strict';
+
```

Comment:
I'm not sure what our conventions are on documentation but it might be helpful to add a documentation header here explaining the mapping object, i.e whether the key relates to the source or value.